docs/setup/independent/high-availability.md

```diff
@@ -0,0 +1,328 @@
+# High Availability
+
+## Set up etcd
+
```

Comment:
What about adding a short explanation about why we are suggesting an external etcd cluster?